### PR TITLE
Added a sign out button

### LINKED ISSUE #116: Logout button
The header bar should display a logout button

### PR DESCRIPTION
**Pull request by eanker**
Added a sign out button at the top of the page.

Fixes #116 

(oops, typo in the branch name)

### CODE REVIEW

**Comment by txsmith** [comment]
Is this ready for review? Conflicts should be resolved.

**Comment by eanker** [comment]
Ah, that conflicts are caused by the fact that I merged the rite to frontend. 
Will resolve these issue, this branch is indeed ready for review (it is in that column on waffle, but the label is not right).

File: src/preview-app.html [hunk 1]
```diff
@@ -146,6 +153,7 @@
             </paper-tab>
           </paper-tabs>
         </template>
+        <paper-button raised on-tap="_signout" hidden$="[[_showSignOut(page.page)]]">Sign out</paper-button>
```
Comment (txsmith) [inline src/preview-app.html]: `_showSignOut` is not needed. If the `paper-button` is moved inside the `template`, it will hide when the current page is the home page.

File: src/preview-app.html [hunk 2]
```diff
@@ -200,10 +208,18 @@
...
+        window.location = '/';
+      }.bind(this));
+    },
     _setToken: function(e) {
       this.$.authenticatedAjax.storeToken(e.detail);
       this.set('page.page', 'projects');
       this.$.ajaxGithubUser.generateRequest();
+    },
+    _showSignOut: function(page) {
+      return page === '';
```
Comment (txsmith) [inline src/preview-app.html]: This can be removed id `paper-button` is moved inside `template`

File: src/preview-app.html (hunk 1 above)
Comment (txsmith) [inline src/preview-app.html]: I would prefer if the button is not `raised`, but instead changes color on hover.

**Review by txsmith — CHANGES_REQUESTED** [review]
Confirmed working!

**Comment by eanker** [comment]
I added the changes requested in the review.
Altough they do not show as outdated, I don't know how that works with this new sort of comments.
